pkg/pocket/core/execute.go

```diff
@@ -60,14 +60,14 @@ func (c *Core) executeByID(id int, sql *types.SQL) {
 // and the manipulated db should be c.dbname
 func (c *Core) coreInitDatabaseExecute(sql *types.SQL) error {
 	switch c.cfg.Mode {
-	case "single":
+	case "single", "tiflash":
 		return errors.Trace(c.coreExec.GetConn().Exec(sql.SQLStmt))
-	case "binlog", "tiflash":
+	case "binlog", "tiflash-binlog":
```

Comment:
what's the meaning of tiflash-binlog?

Reply:
I think it's for CDC, binlog means write from upstream and compare it with downstream.

Reply:
Just apply the same naming pattern for the cdc case. Since it uses `binlog` name as well